clab/config.go

```diff
@@ -569,3 +595,32 @@ func resolveBindPaths(binds []string) error {
 	}
 	return nil
 }
+
+// convertEnvs convert env variables passed as a map to a list of them
+func convertEnvs(m map[string]string) []string {
+	s := make([]string, 0, len(m))
+	for k, v := range m {
+		s = append(s, k+"="+v)
+	}
+	return s
+}
+
+// merge maps m1, m2 and return a resulting map
+func mergeStringMaps(m1, m2 map[string]string) map[string]string {
+	if m1 == nil {
+		return m2
+	}
+	if m2 == nil {
+		return m1
+	}
+	// make a copy of a map
+	m := make(map[string]string)
+	for k, v := range m1 {
+		m[k] = v
+	}
+
+	for k, v := range m2 {
+		m[k] = v
+	}
+	return m
+}
```

Comment:
minor, but there is no need to create a new map, m2 can be merged into m1 and return m1.

Reply:
yes, that was v1 of that function, but then I thought it would make sense to make this func a bit more generic for cases when modifying one of the maps is not desirable

Reply:
ignore then